### PR TITLE
sort partial data

### PR DESCRIPTION
**Pull request by li-qiang**
set  `setSortFn` and sort partial data by it

### CODE REVIEW

File: addon/components/ember-table.js [hunk 1]
```diff
@@ -138,20 +140,15 @@ StyleBindingsMixin, ResizeHandlerMixin, {
...
   },
 
-  updateAntiscroll: function() {
-    this.$('.antiscroll-wrap').antiscroll().data('antiscroll').rebuild();
-    if (this.get('columnsFillTable')) {
-      return this.doForceFillColumns();
-    }
-  },
-
   height: Ember.computed.alias('_tablesContainerHeight'),
```
Comment (jefflage) [inline addon/components/ember-table.js]: Is this an out-of-the-box ember-table function?  Or is it something that you added and no longer need?

File: addon/components/ember-table.js [hunk 2]
```diff
@@ -138,20 +140,15 @@ StyleBindingsMixin, ResizeHandlerMixin, {
     sortByColumn: function(column){
       var sortFn = column.sortFn();
       if(sortFn){
-        this.get('content').sort(sortFn);
+        var content = this.get('content');
+        this.sendAction('setSortFn', column);
```
Comment (jefflage) [inline addon/components/ember-table.js]: What are you doing with this action?  The name is not clear to me what it's intent is.

File: addon/components/ember-table.js (hunk 1 above)
Comment (li-qiang) [inline addon/components/ember-table.js]: `updateAntiscroll ` is extracted from `updateLayout` and `updateLayout` is an existing function in ember table . We can call `updateLayout` instead, so it is't needed.

File: addon/components/ember-table.js (hunk 2 above)
Comment (li-qiang) [inline addon/components/ember-table.js]: The `setSortFn` is used for noticing controller to change sort condition.  When ember table load partial data and someone click the column header to sort, ember table will send action of `setSortFn`. Controller will receive the action and set the sort condition.

**Pull request merged**